doc/src/release_notes.rst

```diff
@@ -91,7 +91,9 @@ pgRouting 3.0.3 Release Notes
 
```

Comment:
The issue is fixed on 3.1.1 so you need to add the #1640 & #1733    also on the section for 3.1.1

Reply:
Once you make the commit, port it to 3.2
That will not be needed to mention on the 3.2 release notes because it is fixed in the past